### DISCUSSION TITLE
Does this app require the cpupower set of utilities?

### DISCUSSION THREAD

**Discussion by bound-variable** [AdnanHodzic/auto-cpufreq#367] [Q&A] [answered]
The man page for cpupower can be found [here](https://man.archlinux.org/man/community/cpupower/cpupower.1.en).

**Comment by AdnanHodzic** [comment, marked as answer]
No, all tools auto-cpufreq needs it will install as its dependencies. It will also detect if some tools which cause problems are installed (like TLP), more info on [README page](https://github.com/AdnanHodzic/auto-cpufreq#why-do-i-need-auto-cpufreq).
